packages/lib/src/spatial-navigation/components/virtualizedList/helpers/getRange.ts

```diff
@@ -37,12 +41,23 @@ const getRangeWithoutFloatHandling = ({
     );
   }
 
-  const halfNumberOfItemsNotVisible = numberOfItemsNotVisible / 2;
+  if (
+    scrollBehavior === 'jump-on-scroll' &&
+    numberOfRenderedItems < 2 * numberOfItemsVisibleOnScreen + 1
+  ) {
+    throw new Error(
```

Comment:
this is a breaking change, would it be possible to just log an error for now and let the dev have a buggy list?
someone could not respect this condition and experience crashes after updates without knowing?

we could throw the error when we do the other breaking change I think?